modules/virt-configuring-masquerade-mode-cli.adoc

```diff
@@ -31,14 +31,19 @@ spec:
       interfaces:
         - name: red
           masquerade: {} <1>
-          ports:
-            - port: 80 <2>
+          ports: <2>
+            - port: 80
   networks:
   - name: red
     pod: {}
 ----
 <1> Connect using masquerade mode
-<2> Allow incoming traffic on port 80
+<2> Optional: List of ports to be exposed from the virtual machine, each specified by the `port` field. The `port` value must be a number between 0 and 65536. When the `ports` array is not used, all ports in the valid range are open to incoming traffic. In the above example, incoming traffic is allowed on port 80.
```

Comment:
It might be worth rephrasing this to start with a verb (parallel structure w/ other callouts). Something like:
`Optional: List the ports that you want to expose from the virtual machine...`

IIRC, "above" is not ideal for accessibility, so I would also s/In the above example/In this example

Reply:
Edited per your suggestion.